include/AddressBook/ABPerson.h

```diff
@@ -26,7 +26,7 @@
 
 typedef uint32_t ABPersonSortOrdering;
 typedef uint32_t ABPersonCompositeNameFormat;
-typedef enum ABPersonImageFormat;
+typedef enum { kABPersonImageFormatThumbnail = 0, kABPersonImageFormatOriginalSize = 2 } ABPersonImageFormat;
```

Comment:
That was what the clang formatter did

---
In reply to: [73208203](https://github.com/Microsoft/WinObjC/pull/786#discussion_r73208203) [](ancestors = 73208203)